src/display/editor/freetext.js

```diff
@@ -58,12 +59,15 @@ class FreeTextEditor extends AnnotationEditor {
 
   static _defaultFontSize = 10;
 
-  static _keyboardManager = new KeyboardManager([
-    [
-      ["ctrl+Enter", "mac+meta+Enter", "Escape", "mac+Escape"],
-      FreeTextEditor.prototype.commitOrRemove,
-    ],
-  ]);
+  static get _keyboardManager() {
+    const km = new KeyboardManager([
```

Comment:
nit: I guess we can survive without having the `km` variable.

Reply:
Sure, that works for me!